src/main/default/lwc/lookup/lookup.js

```diff
@@ -165,12 +165,17 @@ export default class Lookup extends LightningElement {
         this.dispatchEvent(new CustomEvent('selectionchange'));
     }
```

Comment:
After selection, 'No results' listbox is appeared;

<img width="691" alt="win10_20200625_default_1593061961634_89063__Running_" src="https://user-images.githubusercontent.com/5117618/85668550-d1ba9780-b6f9-11ea-8c06-b18bd6ad9323.png">


Because the blur was cancelled by mousedown, `this.hasFocus` boolean will not be off  in `handleResultClick` function.

So it seems to need to add `this.hasFocus = false` in `handleResultClick `.

Reply:
Sorry, I selected wrong line numbers.
`lookup.js` lines 161 to 165.

Reply:
Thanks for raising that.

FYI the problem wasn't the focus, it was the fact that I wasn't clearing the search input correctly.
I was clearing `this.searchTerm` but I also needed to clear `this._cleanSearchTerm`.
`getDropdownClass()` was checking for `this._cleanSearchTerm` to show/hide the dropdown.